src/migrations/20171129064057_multicontainer.js

```diff
@@ -0,0 +1,311 @@
+
+var defaultLegacyVolume = function (appId) {
+	return `resin-app-${appId}`
```

Comment:
This shouldn’t matter since I understand you’ll be always loading the value from the local db, but the cli and builder name the volume for legacy apps as “resin-data”. IMO maintaining consistency is good, but up to you.

Reply:
Hmm, in fact I think it matters — won’t the supervisor try to remove and recreate this volume if it’s given a different value on first api checkin?

Reply:
Yeah, good catch, fixed. Does everything else looks like the default composition?

Reply:
Yep :+1: